internal/services/netapp/netapp_volume_resource.go

```diff
@@ -285,6 +285,22 @@ func resourceNetAppVolume() *pluginsdk.Resource {
 				Optional: true,
 				Default:  false,
 			},
+
+			"encryption_key_source": {
```

Comment:
I'm settting this to computed = true, this is not picking from account level, they're distinct because within an account we can volumes that are encryted using Platform Managed Key (Microsoft.NetApp) of Customer Managed Keys (Microsoft.KeyVault), the default is the former and with the updated SDK version it will return Microsoft.NetApp if this is not set.

I ran two accTests, the basic one and the one with encryption and both passed now.